### PR TITLE
Release v1.0.3

### PR DESCRIPTION
**Pull request by poissoncorp**
https://issues.hibernatingrhinos.com/issue/RavenDB-18848/Helm-Allow-to-use-a-single-TCP-port-for-all-nodes

### CODE REVIEW

File: charts/ravendb-cluster/README.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ ingressClassName: nginx
 # these values are generated by the script and shouldn't be changed
 nodes:
   - nodeTag: a
-    publicTcpPort: 38881
+    publicTcpPort: 443
```
Comment (gregolsky) [inline charts/ravendb-cluster/README.md:53]: Please make a ticket to remove that next time around.

File: charts/ravendb-cluster/misc/generate-values-from-package.py [hunk 2]
```diff
@@ -92,36 +92,36 @@ def get_cluster_info_from_setup_package(
...
-    for file_name in os.listdir("./package"):
-        if not os.path.isdir(f"./package/{file_name}"):
+    for file_name in os.listdir("./setup_package"):
+        if not os.path.isdir(f"./setup_package/{file_name}"):
             continue
-        with open(f"./package/{file_name}/settings.json") as settings_json_ref:
+        with open(f"./setup_package/{file_name}/settings.json") as settings_json_ref:
             settings = json.loads(settings_json_ref.read())
-            port = settings["PublicServerUrl.Tcp"].split(":")[-1]
+            #port = settings["PublicServerUrl.Tcp"].split(":")[-1]
```
Comment (gregolsky) [inline charts/ravendb-cluster/misc/generate-values-from-package.py]: remove?

File: charts/ravendb-cluster/misc/generate-values-from-package.py [hunk 3]
```diff
@@ -92,36 +92,36 @@ def get_cluster_info_from_setup_package(
...
             node_tag = settings["PublicServerUrl.Tcp"].split("//")[1][0]
-            node_infos.append(NodeInfo(node_tag, port))
+            node_infos.append(NodeInfo(node_tag)) #, port))
```
Comment (gregolsky) [inline charts/ravendb-cluster/misc/generate-values-from-package.py]:
```suggestion
            node_infos.append(NodeInfo(node_tag))
```

File: charts/ravendb-cluster/README.md (hunk 1 above)
Comment (poissoncorp) [reply charts/ravendb-cluster/README.md:53]: https://issues.hibernatingrhinos.com/issue/RavenDB-18918/Helm-Remove-publicTcpPort-field-from-valuesyaml

**Pull request merged**